docs/product-user-guide/src/main/asciidoc/business-central-home-con.adoc

```diff
@@ -2,61 +2,162 @@
+not be functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+For more information on Red Hat Technology Preview features support scope, 
+see https://access.redhat.com/support/offerings/techpreview/. 
+====
+
+The Deployments perspective enables you to provision projects on OpenShift. In this release, it is hidden by default. Perform the following steps to enable it:
+
+. Go to *Settings* → *Roles*.
+. Select the *admin* role.
+. In the Permissions section, click the arrow next to Perspectives.
+. Click the trash icon next to Deployments. The DevOps → *Deployments* link is now available on the Home page.
+
+* Execution Servers
+The Execution Servers perspective contains a list of the deployed Intelligent Process Server templates and containers associated with the templates. This perspective is visible to users with the following roles:
+
+** Admin
+** Developer
+
+.Manage
+The Manage group contains five perspectives:
+
+* Process Definitions
++
+The Process Definitions perspective contains a list of deployed process definitions. It enables you to instantiate and manage the deployed processes. This perspective is visible to users with the following roles:
+
+** Admin
+** Analyst
+** Developer
+** User
+** Process-admin
+
+* Process Instances
 +
-The view is available for users with the `admin` role only.
+The Process Instances perspective contains a list of the instantiated processes. It enables you to view their execution work flow and history. This perspective is visible to users with the following roles:
+
+** Admin
+** Analyst
+** Developer
+** User
+** Process-admin
+
+* Task Administration
 +
-** *Administration* perspective contains:
-*** The *File Explorer* view with available asset repositories.
-*** The editor area on the right of the *File Explorer* view, where the respective editor appears when a resource is opened.
+The Task Administration perspective contains a list of all manually created tasks, created by all users. This perspective is visible to users with the following roles:
+
+** Admin
+** Process-admin
+
+* Jobs
 +
-The *Administration* perspective enables an administrator to connect a Knowledge Store to a repository with assets and to create a new repository. For more information, see the _{ADMIN_GUIDE}_.
+The Jobs perspective enables you to monitor and trigger asynchronous jobs scheduled for the Executor Service. This perspective is visible to users with the following roles:
+
+** Admin
+** Analyst
+** Developer
+
+* Execution Errors
++
+The Execution Errors perspective lists any errors that occur during jBPM process instances or jBPM executor service jobs on a KIE server. Use this perspective to help troubleshoot issues. This perspective is visible to users with the following roles:
```

Comment:
We usually avoid using the terms "jBPM" or "drools" in the product docs as they are community names (unless an artifact name contains jbpm).  We are supposed to use BPM Suite instead of jBPM and BRMS instead of drools.
However, in this case, we can simply call them plain 'executor service' and 'process instances'.